### ISSUE TITLE
Homework for STAT547

### ISSUE THREAD

**Issue by dtavern** [STAT545-UBC/Discussion#410]
@jennybc 

I remember you mentioning that we choose a certain number (2?) of assignments past #06 and that we are not required to complete them all. Could you (or any of the TAs) confirm if I am remembering correctly?

Thanks!

**Comment by jennybc** [comment]
Yes. Everyone should do the first one = HW06. Then I will take the highest two marks of the remaining four. Which implies you can do 2 and skip 2.

**Comment by philstraforelli** [comment]
Do we need to announce if we choose to skip a homework? Or would it just be implicit in that we're not opening an issue to announce the homework is done?

**Comment by jennybc** [comment]
Announcing it would be sort fun, eh? 😃

However, I say you can simply NOT open an issue for that HW.
